src/darkslide/utils.py

```diff
@@ -26,15 +26,11 @@ def encode_data_from_url(url, source_path):
     real_path = url if os.path.isabs(url) else os.path.join(source_path, url)
 
     if not os.path.exists(real_path):
-        print('%s was not found, skipping' % url)
```

Comment:
I don't see why these need to be removed...

Reply:
I think it should be removed, for several reasons. All the messages printed out by _darkslide_ are issued by a "logger" at a higher level either in _macro.py_ or _generator.py_ - not by a direct `print()`, and not by utility functions.

But most importantly, this function `encode_data_from_url()` is called several times in a "trial and error" way by `embed_url_data()`, which tries to find the files in several directories (see `css_dirs`). For every directory where the file is _not_ found, you would get a message `yourfile was not found, skipping`... Whereas in the end, the file may very well be found in one of the last directories.